### PR TITLE
lfx term 3: add a second CRI-O project

### PR DESCRIPTION
**Pull request by haircommander**

### CODE REVIEW

**Comment by haircommander** [comment]
cc @saschagrunert @sohankunkerkar

File: programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md [hunk 1]
```diff
@@ -42,6 +42,14 @@
   - Sascha Grunert, saschagrunert, sgrunert@redhat.com
 - Upstream Issue (URL): https://github.com/containers/conmon-rs/issues/1126
```
Comment (nate-double-u) [inline programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md:44]: We should add a project title:

```suggestion

#### CRI stats KEP

```

**Review by nate-double-u — APPROVED** [review]
Thanks @haircommander! Just one note in line:

File: programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md [hunk 2]
```diff
@@ -42,6 +42,14 @@
...
 
+- Description: [CRI stats KEP](https://github.com/kubernetes/enhancements/issue/2371) is an effort to take the container stats and metrics collection from cAdvisor and move it to the CRI implementations. CRI-O will soon have support for stats and metrics collected through CRI, but work needs to be done to verify and validate these fields, and make sure their collection is performant as possible.
```
Comment (nate-double-u) [inline programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md:45]: This link 404s

**Review by saschagrunert — APPROVED** [review]

File: programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md (hunk 2 above)
Comment (saschagrunert) [inline programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md:45]:
```suggestion
- Description: [CRI stats KEP](https://github.com/kubernetes/enhancements/issues/2371) is an effort to take the container stats and metrics collection from cAdvisor and move it to the CRI implementations. CRI-O will soon have support for stats and metrics collected through CRI, but work needs to be done to verify and validate these fields, and make sure their collection is performant as possible.
```

**Comment by haircommander** [comment]
thanks y'all!

**Pull request merged**